cmd/servicelog/common.go

```diff
@@ -73,68 +71,51 @@ func applyFilters(ocmClient *sdk.Connection, filters []string) ([]*v1.Cluster, e
 	return items, err
 }
 
-func sendRequest(request *sdk.Request) (*sdk.Response, error) {
-	response, err := request.Send()
-	if err != nil {
-		return nil, fmt.Errorf("cannot send request: %q", err)
+func check(response *ocm_servicelog.ClusterLogsAddResponse, clusterMessage servicelog.Message) {
```

Comment:
Can this function name be made more clear? What is being checked here?

Reply:
Let's read and suggest here, sec

Reply:
aggregateFailedResponses